content/blog/2018-04-23-noo-noo-from-collyers-college-wins-sourcebots-2018.md

```diff
@@ -0,0 +1,86 @@
+
+The matches were broken into two main sections. First was a league stage with
+matches spaced throughout the weekend and interspersed with "tinker time" which
+both allowed the teams to test and improve their robots as well as gauge the
+strengths of their opponents. The time between rounds proved invaluable to many
+of the teams and we saw the robots improve greatly over the course of the
+weekend.
+
+Once the league was complete, the results from the league seeded a knockout
+stage. While every robot entered the knockouts, it all came down to the final
+match between _[Noo Noo][CLY-robot-image]_ from [Collyer's][collyers] team
+_SCC_, _[Out Of The Blue][RGS-robot-image]_ from [RGS Guildford][rgs-guildford],
+_[Brenda][BPV2-robot-image]_ from [Barton Peveril][barton-peveril]'s second team
+and [the robot][BRK-robot-image] from team _Bodgenhurst_ from
+[Brockenhurst][brockenhurst].
+
+[barton-peveril]: https://www.barton-peveril.ac.uk/
+[brockenhurst]: https://www.brock.ac.uk/
+[collyers]: http://www.collyers.ac.uk/
+[rgs-guildford]: https://www.rgs-guildford.co.uk/
+
+[BPV2-robot-image]: /img/2018/robots/BPV2.jpg
+[BRK-robot-image]: /img/2018/robots/BRK.jpg
+[CLY-robot-image]: /img/2018/robots/CLY.jpg
+[RGS-robot-image]: /img/2018/robots/RGS.jpg
+
+The final promised to be interesting as a variety of different approaches were
+taken by the participating robots. _Out Of The Blue_, _Brenda_ and
+_Bodgenhurst_'s robot each attempted to capture tokens in their open front
+halves by driving up to tokens with their arms open, before closing them around
+the tokens.
+
+_Out Of The Blue_ extended this strategy by first driving through the tokens in
+their own zone with their arms closed, which pushed some of the undesirable
+tokens out of their zone before they collected their own tokens. This had given
+the robot a strong position after the league matches; by ranking second they
+went straight through to the semi-finals.
+
+In the end however it was _Noo Noo_'s great use of its [Mecanum
+wheels][mecanum-wheels] and long grabber arm to drive _sideways_ through their
+own zone at the start of the match, sweeping the undesirable tokens out of the
+zone, which granted their victory.
+
+[mecanum-wheels]: https://en.wikipedia.org/wiki/Mecanum_wheel
+
+### Prizes
+
+| Prize                 | Team                            |
+| --------------------- | ------------------------------- |
+| 1<sup>st</sup> Place  | Collyer's — SCC                 |
+| 2<sup>nd</sup> Place  | RGS Guildford — Out Of The Blue |
+| 3<sup>rd</sup> Place  | Brockenhurst — Bodgenhurst      |
+| Most Ambitious        | RGS Guildford — Out Of The Blue |
+| Best Looking          | Barton Peveril — Orion          |
+
+### Livestream
+
+This year's knockout matches were streamed live on Facebook by [SUSU TV][susu-tv]
+who did an amazing job, which included setting up a small studio at the venue
+and interviewing some of the competitors.
```

Comment:
I like this phrasing better, though the flow still seems a little off. How about rearranging like so:

> This year's knockout matches were streamed live on Facebook by [SUSU TV][susu-tv], which included setting up a small studio at the venue and interviewing some of the competitors. They did an amazing job and we look forward to collaborating with them again in the future.

I'm worried the newly added clause ("we look forward to...") may be a little presumptuous, so feel free to swap this out if you like.

Reply:
I agree the phrasing here is better (mostly by being two sentences), though I agree with the concern about the collaboration part.
I'm going to leave this as-is.